seed/seed.js

```diff
@@ -5,63 +5,169 @@
 const childProcess = require('child_process');
 const path = require('path');
 
+const { getAllCourses } = require('../resources/grabs');
 const models = require('../server/db/index.js');
 const { db } = models;
 const userList = require('./seedlings/userSeed');
-const groupList = require('./seedlings/groupSeed');
+const { groupList } = require('./seedlings/groupSeed');
 const eventList = require('./seedlings/eventSeed');
 const postList = require('./seedlings/postSeed');
+const zipList = require('./seedlings/zipSeed');
+
+const findModel = async (name) => {
+  const topic = await models.Topic.findOne({
+    where: {
+      name
+    }
+  })
+  return topic;
+}
+
+const hasKeys = (obj) => {
+  let count = 0
+  for (let key in obj) {
+    if (key) count++
+  }
+  if (count >= 1) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+const topicNameIds = async (modelArr) => {
+  try {
+    const topicAssociation = {};
+    modelArr.forEach(async t => {
+      try {
+        const top = await t
```

Comment:
Why are we awaiting here? It looks like you have already done that in findModel.